frontend/src/views/chart/view/ChartEdit.vue

```diff
@@ -467,7 +467,11 @@ export default {
   },
   watch: {
     'param': function() {
-      this.getData(this.param.id)
+      if(this.param.optType === 'new'){
```

Comment:
Codacy has a fix for the issue: [Expected space(s) after "if".](https://app.codacy.com/gh/dataease/dataease/pullRequest?prid=7710993)
```suggestion
      if (this.param.optType === 'new'){
```